Week3/StringSeg.java

```diff
@@ -0,0 +1,68 @@
+import java.util.HashSet;

```

Comment:
Keep an eye on consistent spacing. Examples by line
various) consistency with new line before open braces
8-17) you have inconsistent spacing around == throughout these cases
22) your other one line if statements use braces, add them here
38) extra space in for loop
49) extra space before ;
40-44, 48-51) tab depth is off